fv3core/stencils/remap_profile.py

```diff
@@ -516,19 +516,22 @@ def compute(
     km: int,
     i1: int,
     i2: int,
+    j1: int,
+    j2: int,
     iv: int,
     kord: int,
-    jslice: Tuple[int],
     qmin: float = 0.0,
 ):
     assert kord <= 10, f"kord {kord} not implemented."
-
+    grid = spec.grid
+    km = grid.npz
     i_extent: int = i2 - i1 + 1
-    j_extent: int = jslice.stop - jslice.start
-    js: int = jslice.start
-    orig: Tuple[int] = (i1, js, 0)
-    full_orig: Tuple[int] = (spec.grid.is_, js, 0)
+    j_extent: int = j2 - j1 + 1
+
+    orig: Tuple[int] = (i1, j1, 0)
+    full_orig: Tuple[int] = grid.compute_origin()
```

Comment:
I would think js --> j1 rather than grid.js, but looks like this is only ever used to set the origin of temporary storages, so compute_domain makes a lot of sense